test/e2e-test/src/index.ts

```diff
@@ -172,26 +179,90 @@ class Demo {
     Demo.assertField(member.name, response, "crv", "X25519");
     Demo.assertField(member.name, response, "kty", "OKP");
 
-    const _ = await Api.keyInitial(
+    console.log(`📝 Get initial key...`);
+    let keyResponse = await Api.key(
+      this.demoProps,
+      member,
+      JSON.stringify(attestation),
+      202,
+      false,
+      this.createHttpsAgent(member.id, AuthKinds.JWT),
+      access_token
+    ).catch((error) => {
+      console.log(`keyInitial error: `, error);
+      throw error;
+    });
+    if (!keyResponse) {
+      console.log(
+        `✅ [PASS] - Initial key response must be undefined`,
+      );
+    } else {
+      throw new Error(
+        `🛑 [TEST FAILURE]: Initial key response must be undefined`,
+      );
+    }
+
+    console.log(`📝 Get initial key-Bad request...`);
+    let keyBadResponse = await Api.key(
+      this.demoProps,
+      member,
+      JSON.stringify(''),
+      400,
+      false,
+      this.createHttpsAgent(member.id, AuthKinds.MemberCerts)
+    ).catch((error) => {
+      console.log(`keyInitial error: `, error);
+      throw error;
+    });
+
+    if ((<any>keyBadResponse).error.message === "missing attestation") {
+      console.log(
+        `✅ [PASS] - Missing attestation error found`,
+      );
+    } else {
+      throw new Error(
+        `🛑 [TEST FAILURE]: Initial key response message failed`
+      );
+    }
+  
+    console.log(`📝 Get initial key-No auth...`);
+    keyBadResponse = await Api.key(
       this.demoProps,
       member,
       JSON.stringify(attestation),
+      401,
+      false,
+      this.createHttpsAgent(member.id, AuthKinds.NoAuth)
     ).catch((error) => {
-      console.log(`keyInitial error: ${error?.response?.data?.nessage}`);
+      console.log(`keyInitial error: `, error);
       throw error;
     });
 
+    if ((<any>keyBadResponse).error.message === "Invalid authentication credentials.") {
+      console.log(
+        `✅ [PASS] - Missing attestation error found`,
+      );
+    } else {
+      throw new Error(
+        `🛑 [TEST FAILURE]: Initial key response message failed`
+      );
+    }
+  
+  
     // Wait for receipt to be generated
-    await Demo.sleep(5000);
+    await Demo.sleep(3000);
```

Comment:
Maybe it's worth to remove these sleep now to avoid future pain (in a different PR)?
I think we can just retry getting wrapped key while getting 202.